src/pages/CookieClicker.vue

```diff
@@ -1,49 +1,331 @@
-            </figure>
-        </div>
-        <div class="column is-6 has-background-link">
-            asdas
-        </div>
-        <div class="column is-2 has-background-warning">
-            <button v-for="building in buildings" :disabled="cookies<building.price" @click="buyBuilding(building)" class="button is-primary is-large is-fullwidth">
-                {{ building.name }} 🍪{{ building.price }} #{{ building.count }}
-            </button>
+  <div class="columns is-gapless main-container">
+    <!-- Left Column -->
+    <div class="column is-4 has-text-centered p-5 left-panel">
+      <h1 class="is-size-1 has-text-white">{{ cookies.toFixed(1) }} cookies</h1>
+      <h3 class="is-size-3 has-text-light">{{ cps.toFixed(1) }} cps</h3>
+
+      <div class="cookie-container" @click="cookies++">
+        <img
+          class="cookie-img"
+          src="https://sweetlorens.com/cdn/shop/products/Copy-of-Chocolate-Chunk-Full-Cookie-transparent-background.png?v=1687811511"
+          alt="Main Cookie"
+        />
+      </div>
+    </div>
+
+    <!-- Middle Column -->
+    <div class="column is-6 middle-panel">
+      <div class="has-text-centered p-6 is-size-4 has-text-light">
+        <p>Welcome to Cookie Clicker Deluxe 🍪</p>
+        <p>Golden Cookies appear randomly – click fast!</p>
+
+        <div class="mt-6">
+          <h3 class="has-text-white is-size-5 mb-2">🏆 Achievements</h3>
+          <ul>
+            <li v-for="a in achievements" :key="a" class="has-text-warning">
+              {{ a }}
+            </li>
+          </ul>
+
+          <h3 v-if="upgrades.length" class="has-text-white is-size-5 mt-5 mb-2">🧠 Upgrades</h3>
+          <ul>
+            <li v-for="u in upgrades" :key="u" class="has-text-info">
+              {{ u }}
+            </li>
+          </ul>
         </div>
+      </div>
+    </div>
+
+    <!-- Right Column -->
+    <div class="column is-2 building-panel p-3">
+      <button
+        v-for="building in buildings"
+        :key="building.name"
+        :disabled="cookies < building.price"
+        @click="buyBuilding(building)"
+        class="button is-success is-fullwidth mb-3 building-button"
+      >
+        {{ building.name }} 🍪{{ building.price }} #{{ building.count }}
+      </button>
+    </div>
+
+    <!-- Golden Cookie -->
+    <div
+      v-if="goldenCookieVisible"
+      :style="{
+        position: 'absolute',
+        top: goldenCookiePosition.top,
+        left: goldenCookiePosition.left,
+        zIndex: 9999,
+      }"
+      @click="clickGoldenCookie"
+      class="golden-cookie"
+    >
+      <img
+        src="https://pngimg.com/uploads/cookie/cookie_PNG13657.png"
+        alt="Golden Cookie"
+        class="golden-cookie-img"
+      />
     </div>
-</template>
\ No newline at end of file
+
+    <!-- Toast Notification -->
+    <transition name="toast-fade">
+      <div v-if="showToast" class="toast">
```

Comment:
Its fine that you have built your own toast, but you could also pull in https://github.com/rfoel/bulma-toast?tab=readme-ov-file (yes the demo page is broken idk seems small bug for them but the package should work)